src/components/pages/Market.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { ContractHound, HoundInfo, houndInfoFromGenome } from "../../scripts/hound-genome";
+import { TzContext } from "../context/TzToolKitContext";
+import HoundCard from "../HoundCard";
+import { BigNumber } from "bignumber.js";
+import "../css/Market.css";
+
+type HoundListEntry = {
+  id: ContractHound;
+  price: BigNumber;
+  hound: HoundInfo;
+};
+
+type HoundList = Array<HoundListEntry>;
+
+function HoundList(): JSX.Element {
+  const tzContext = React.useContext(TzContext);
+  const mapOfHoundsOnSale: Map<string, ContractHound> = tzContext.contractStorage.market.valueMap;
```

Comment:
![Screenshot 2021-10-10 182121](https://user-images.githubusercontent.com/51472181/136697710-a5b6acba-d2db-4020-aa9b-c9d88f0b6580.png)
await `tzContext.loadContract()` will be needed to keep check when page is refreshed.